.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+## ClickUp
```

Comment:
"ClickUp" sounds more like an internal detail for an OSS project. May be the thing we care about is generally any issue tracking reference ?

Reply:
I've changed the wording, what do you think about it now?